test/models/test_mnasnet.py

```diff
@@ -1,28 +1,16 @@
 import numpy as np
 import pytest
 from test.utils import convert_and_test
-from torchvision.models import mnasnet0_5, mnasnet1_0, mnasnet0_75, mnasnet1_3
+from torchvision.models import mnasnet0_5, mnasnet0_75
```

Comment:
why did we keep here both mnasnet0_5 and mnasnet0_75 and removed the other two?

Reply:
I kept only the 0_5 in a later commit, but forgot to remove the import. will do now.

Reply:
I agree, i ask because i think it can improve other ci tests that we have that dont use that mechanism